public/resources/stylesheets/page/_formVerify.scss

```diff
@@ -0,0 +1,221 @@
+.formUser {
+    height: 50em;
+    width: 100em;
+    display: flex; 
+    box-shadow: $box-shadow;
+    overflow: hidden;
+    border: 3px solid $tone_wl;
+    border-radius: 30px;
+    backdrop-filter: blur(20px);
+    animation: scale-in-center 0.5s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
+}
+
+.leftImg {
+    @include flex-center;
+    flex-direction: column;
+    width: 50%;
+    background: $tone_wl;
+    backdrop-filter: blur(30px);
+}
+
+.rightForm {
+    border-radius: 0 0.9rem 0.9rem 0;
+    position: relative;
+    width: 50%;
+    padding: 20px;
+    overflow: hidden;
+    backdrop-filter: blur(30px);
+    background: $tone_wl;
+    
+}
+
+.imgPass {
+    width: 600px;
+}
+
+.imgNewPass {
+    width: 500px;
+}
+
+.titleForm {
+    font-size: 47px;
+    color: $brow1;
+    margin: 20px 0 20px 0;
+    font-weight: 600;
+    color: $font_w;
+}
+
+.primaryForm {
+    padding: 1vw 4vw; 
+    border-radius: 2rem;
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    transition: .3s;
+    height: 90%;
+    justify-content: center;
+    text-align: center;
+}
+
+.form-inputs {
+    Width: 100%;
+}
+
+.instruction {
+    color: $font_w;
+    width: 100%;
+    font-size: 19px;
+    margin: 0 0 20px 0;
+
+    span {
+        font-weight: 800;
+    }
+}
+
+.confirmEmail {
+    border: none;
+    @include flex-center;
+    height: 2.8rem;
+    border-radius: 0.5rem;
+    font-size: 1em;
+    width: 100%;
+
+    cursor: pointer;
+    font-weight: 600;
+    color: $tone_gH;
+    background-color: $tone_gre;
+
+    &:hover {
+        background-color: $tone_gH;
+        color: $tone_gre;
+    }
+}
+
+.formOption {
+    gap: 10px;
+    color: $font_w;
+    font-size: 1.1rem;
+    margin-top: 30px;
+    @include flex-center;
+    font-weight: 600;
+}
+
+.optionUser {
+    margin-left: 0.5rem;
+    cursor: pointer;
+    text-decoration: none;
+    padding: 0.5rem;
+    border-radius: 0.5rem;
+    color: $tone_gH;
+    background-color: $tone_gre;
+
+    &:hover {
+        background-color: $tone_gH;
+        color: $tone_gre;
+
+    }
+}
+
+.optionResend {
+    margin-left: 0.5rem;
+    cursor: pointer;
+    text-decoration: none;
+    padding: 0.7rem 1rem;
+    border-radius: 1.5rem;
+    color: $font_w;
+    font-weight: 600;
+    background-color: $tone_gH;
+    
+
+    &:hover {
+        text-decoration: none;
+        font-weight: 600;
+        color: $tone_gre;
+    }
+}
+
+.formOption2 {
```

Comment:
you could improve the naming, it is a must in all developments teams, so you could use this time as a practices to be ready for the real professional work